src/main/java/pl/rejmanbeata/bakery/jpa_repository/ClientRepository.java

```diff
@@ -0,0 +1,7 @@
+package pl.rejmanbeata.bakery.jpa_repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import pl.rejmanbeata.bakery.database.ClientEntity;
+
+public interface ClientRepository extends JpaRepository<ClientEntity, Long> {
```

Comment:
I would guess we also need this to be managed by Spring and hence we want to add `@Repositroy` annotation:)